src/app/drivingLicence/controllers/licence-issuer.js

```diff
@@ -32,30 +31,22 @@ class LicenceIssuerController extends BaseController {
           );
           return next();
         }
-          case "DVA": {
-            logger.info(
-              "licence-issuer: user selected DVA : redirecting to driving licence details",
-              {
-                req,
-                res,
-              }
-            );
-            return next();
-          }
+        case "DVA": {
+          logger.info(
+            "licence-issuer: user selected DVA : redirecting to driving licence details",
+            {
+              req,
+              res,
+            }
+          );
+          return next();
+        }
       }
       return next(new Error("licence-issuer: Invalid action " + action));
     } catch (err) {
       return next(err);
     }
   }
-
-  next(req) {
```

Comment:
How come we have we removed this in the case of no licence we want to return to core

Reply:
The functionality of the next prototype is moved to steps.js - If noLicence=true, the prove another way step is invoked, other wise the /details step invoked as before.

```
  "/licence-issuer": {
    controller: licenceIssuer,
    fields: ["licenceIssuer"],
    next: [
      {
        field: "noLicence",
        value: true,
        next: "/prove-another-way",
      },
      "/details",
    ],
```